examples/flows/evaluation/eval-qna-non-rag/README.md

```diff
@@ -64,4 +64,33 @@ pf flow test --flow . --inputs metrics="ada_similarity,gpt_fluency,f1_score" que
 pf run create --flow . --data ./data.jsonl --column-mapping question='${data.question}' answer='${data.answer}' context='${data.context}' ground_truth='${data.ground_truth}' metrics='f1_score,gpt_groundedness' --stream
 ```
 You can also skip providing `column-mapping` if provided data has same column name as the flow.
-Reference [here](https://aka.ms/pf/column-mapping) for default behavior when `column-mapping` not provided in CLI.
\ No newline at end of file
+Reference [here](https://aka.ms/pf/column-mapping) for default behavior when `column-mapping` not provided in CLI.
+
+## 3. Run and Evaluate your flow with this Q&A evaluation flow
+After you develop your flow, you may want to run and evaluate it with this evaluation flow. 
+
+Here we use the flow [basic_chat](https://github.com/microsoft/promptflow/tree/main/examples/flows/chat/basic-chat) as the flow to evaluate. It is a flow demonstrating how to create a chatbot with LLM.  The chatbot can remember previous interactions and use the conversation history to generate next message, given a question. 
```

Comment:
```suggestion
Here we use the flow [basic_chat](../../chat/basic-chat/) as the flow to evaluate. It is a flow demonstrating how to create a chatbot with LLM.  The chatbot can remember previous interactions and use the conversation history to generate next message, given a question. 
```

Reply:
I prefer local path than url, feel free to apply or ignore (same below)

Reply:
Changed url to local path.
Thanks!